packages/alpha/src/types.ts

```diff
@@ -17,47 +17,53 @@ import {
 
 export * from '@cognite/sdk';
 
-export interface MonitoringTaskParametersCreate {
+export type ModelExternalId = 'threshold';
+
+export const ModelExternalId = {
```

Comment:
```suggestion
export const MonitoringTaskModelExternalId = {
```
prepend API name for users to get a nice autocomplete